docs/iot/guides/things-api-device-5.0.md

```diff
@@ -0,0 +1,211 @@
+# Device Definition
+
+The device definition is stored in two JSON configuration files such as `master.json` and `resource.json` in the directory of shared/res.
+
+## Master Json File Descriptions
+Master json file can represent device and configuration for thing agent.
+
+* ***devices***[objectArray] : Define device type, name and so on.
+  * ***deviceType***[string] : Device type of thing
+  * ***deviceName***[string] : Human friendly name defined by the vendor(n of oic.wk.d)
+  * ***mnid***[string] : Name of manufacturer(mnmn of oic.wk.p)(Manufacturer Identifier issued by DevWorkSpace)
+  * ***vid***[string] : Vendor defined string for the platform(vid of oic.wk.p)
+  * ***nick***[string] : (Optional) Nick name
+* ***configuration***[object] : Define OTM, certificate type, and easy-setup information.
+  * ***otm***[int] : Ownership Transfer Method: b0001(Random PIN-based), b0010(UserConfirm-based), b0100(Preconfigured PIN). The default value is 2.
+  * ***crtType***[int] : Certificate type: 1(Test Certificate), 2(Artik Certificate), 3(Custom). To use test certificate as described in "Getting Started", the value should be 1.
+  * ***easySetup***[object] : Easy-setup information
+    * ***mode***[string] : Easy-setup mode: 1(SoftAP), 2(Custom). To use Soft AP as described in "Getting Started", the value should be 1.
```

Comment:
Can you please a space after 1...

1 (SoftAP), 2 (Custom). To use SoftAP as described in "Getting Started", the value should be 1.

Please remove the period to maintain consistency.

Reply:
Ok. Done.